src/Forms/EffectKaraoke.cs

```diff
@@ -306,12 +313,20 @@ private void MakeAnimation()
                 {
                     text += tag + c;
                     tag = string.Empty;
+                    // Take next char if current is (\r or \n);
+                    while ((_paragraph.Text[i] == '\r' || _paragraph.Text[i] == '\n') && (i + 1 < _paragraph.Text.Length))
+                    {
+                        text += _paragraph.Text[++i];
+                    }
                     string afterText = string.Empty;
                     if (i + 1 < _paragraph.Text.Length)
                         afterText = _paragraph.Text.Substring(i + 1);
-                    if (string.Compare(afterText, endFontTag, StringComparison.InvariantCultureIgnoreCase) == 0 && afterCurrentTag.StartsWith("<font", StringComparison.InvariantCultureIgnoreCase))
+                    if (afterText.StartsWith(endFontTag, StringComparison.Ordinal) && afterCurrentTag.StartsWith("<font", StringComparison.Ordinal))

```

Comment:
This is was what this line was causing
1
00:00:01,000 --> 00:00:01,285
<font color="#ff0000">(</font><font color="#808080">MUSIC</font>)

2
00:00:01,286 --> 00:00:01,570
<font color="#ff0000">(M</font><font color="#808080">USIC</font>)

3
00:00:01,571 --> 00:00:01,856
<font color="#ff0000">(MU</font><font color="#808080">SIC</font>)

4
00:00:01,857 --> 00:00:02,142
<font color="#ff0000">(MUS</font><font color="#808080">IC</font>)

5
00:00:02,143 --> 00:00:02,428
<font color="#ff0000">(MUSI</font><font color="#808080">C</font>)

6
00:00:02,429 --> 00:00:02,713
<font color="#ff0000">(MUSIC</font><font color="#808080"></font>)

7
00:00:02,714 --> 00:00:03,000
<font color="#ff0000">(MUSIC</font>)</font>


Note: Original text: (<font color="#808080">MUSIC</font>)